pyiron_base/jobs/job/generic.py

```diff
@@ -418,13 +434,13 @@ def refresh_job_status(self):
         """
         Refresh job status by updating the job status with the status from the database if a job ID is available.
         """
-        if self.job_id:
```

Comment:
If you are anyway changing this line, can you add `if self.job_id is not None`?

Reply:
sorry I wanted to comment on the changed line, but you know what I mean.

Reply:
I updated it.